librz/core/disasm.c

```diff
@@ -5893,31 +5853,18 @@ RZ_IPI ut64 rz_core_backward_offset(RZ_NONNULL RzCore *core, RZ_NONNULL RZ_INOUT
 RZ_API int rz_core_print_disasm_instructions(RzCore *core, int nb_bytes, int nb_opcodes) {
```

Comment:
Please convert this function comment to the Doxygen. Moreover, I wonder if instead of using defines, it's better using `static const ut8` for `MAX_OPSIZE`, also do it only once in this file